GameServer/Logic/Packets/Packets.cs

```diff
@@ -832,6 +832,65 @@ public InhibitorAnnounce(Inhibitor inhi, InhibitorAnnounces type, GameObject kil
         }
     }
 
+    public class HideUi : BasePacket
+    {
+        public HideUi() : base(PacketCmdS2C.PKT_S2C_HideUi)
+        {
+
+        }
+    }
+
+    public class UnlockCamera : BasePacket
+    {
+        public UnlockCamera() : base(PacketCmdS2C.PKT_S2C_UnlockCamera)
+        {
+
+        }
+    }
+
+    public class MoveCamera : BasePacket
+    {
+        public MoveCamera(Champion champ, float x, float y, float z, float speed) : base(PacketCmdS2C.PKT_S2C_MoveCamera, champ.getNetId())
+        {
+            // Unk, if somebody figures out let @horato know
+            buffer.Write((byte)0x97);
+            buffer.Write((byte)0xD4);
+            buffer.Write((byte)0x00);
+            buffer.Write((byte)0x58);
+            buffer.Write((byte)0xD7);
+            buffer.Write((byte)0x17);
+            buffer.Write((byte)0x00);
+            buffer.Write((byte)0xCD);
+            buffer.Write((byte)0xED);
+            buffer.Write((byte)0x13);
+            buffer.Write((byte)0x01);
+            buffer.Write((byte)0xA0);
+            buffer.Write((byte)0x96);
+
+            buffer.Write((float)x);
+            buffer.Write((float)y);
+            buffer.Write((float)y);
+            buffer.Write((float)speed); // less = faster
```

Comment:
Might be something like `time` instead of speed. Would explain the less = faster.